### ISSUE TITLE
Cocoapods not working in Xcode 16 macOS 15.0.1

### ISSUE THREAD

**Issue by LuisMoraEL** [CocoaPods/CocoaPods#12625]
――― MARKDOWN TEMPLATE ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

### Command

```
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod init
```

### Report

I tried to create the pod file in order to install frameworks but it didn't work.

### Stack

```
   CocoaPods : 1.15.2
        Ruby : ruby 3.3.5 (2024-09-03 revision ef084cc8f4) [x86_64-darwin24]
    RubyGems : 3.5.16
        Host : macOS 15.0.1 (24A348)
       Xcode : 16.0 (16A242d)
         Git : git version 2.39.5 (Apple Git-154)
Ruby lib dir : /Users/sedeam/.rbenv/versions/3.3.5/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

### Error

```
RuntimeError - `PBXGroup` attempted to initialize an object with unknown ISA `PBXFileSystemSynchronizedRootGroup` from attributes: `{"isa"=>"PBXFileSystemSynchronizedRootGroup", "exceptions"=>["848EFC072CB6F7A8004A4750"], "path"=>"test_pods", "sourceTree"=>"<group>"}`
If this ISA was generated by Xcode please file an issue: https://github.com/CocoaPods/Xcodeproj/issues/new
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:359:in `rescue in object_with_uuid'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:349:in `object_with_uuid'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:300:in `block (2 levels) in configure_with_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:299:in `each'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:299:in `block in configure_with_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:296:in `each'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:296:in `configure_with_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:272:in `new_from_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:350:in `object_with_uuid'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:290:in `block in configure_with_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:287:in `each'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project/object.rb:287:in `configure_with_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:272:in `new_from_plist'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:213:in `initialize_from_file'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:113:in `open'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command/init.rb:41:in `validate!'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/claide-1.1.0/lib/claide/command.rb:333:in `run'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command.rb:52:in `run'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/bin/pod:55:in `<top (required)>'
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod:25:in `load'
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod:25:in `<main>'
```

――― TEMPLATE END ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

**Comment by chenfanfang** [comment]
same

**Comment by erikggupt** [comment]
Same here

**Comment by erikggupt** [comment]
Please follow the below steps to fix this issue

Step 1: Right click project Folder in Xcode and convert to group

Step 2: right click projectname.xcodeproj file and select Show Package Contents
you can see the file project.pbxproj and open with texteditor

Remove the following lines in this file
minimizedProjectReferenceProxies = 1;
preferredProjectObjectVersion = 77;

Also update the below version
objectVersion = 77; to objectVersion = 56;

Now you can pod init and install with your project

**Comment by LuisMoraEL** [comment]
Thanks a lot

**Comment by eomonteiro** [comment]
#12627

**Comment by martg3** [comment]
> Please follow the below steps to fix this issue
> 
> Step 1: Right click project Folder in Xcode and convert to group
> 
> Step 2: right click projectname.xcodeproj file and select Show Package Contents you can see the file project.pbxproj and open with texteditor
> 
> Remove the following lines in this file minimizedProjectReferenceProxies = 1; preferredProjectObjectVersion = 77;
> 
> Also update the below version objectVersion = 77; to objectVersion = 56;
> 
> Now you can pod init and install with your project

The method functions correctly, but issues arise when attempting to run or build the project, preventing it from executing or compiling successfully.

```
error: Sandbox: bash(84265) deny(1) file-write-create Pods/resources-Pods/resources-to-copy-app.txt: Operation not permitted
realpath: illegal option -- m
usage: realpath [-q] [path ...]
:20: error: Unexpected failure
```

**Comment by LuisMoraEL** [comment]
I don't have any problem with this solution, but in my experience, if you have problems with the sandbox and will not use it, you can deactivate the sandbox, first go to your project in Xcode, in your project file go to built settings and make sure you have selected the project and not the target, in built settings - User Scrip Sandboxing change the value to "no", this is a good solution if you are working with Firebase for example, I hope this will help you!.

**Comment by LuisMoraEL** [comment]
"PBXFileSystemSynchronizedRootGroup" to "PBXGroup" is not working 
### Command

```
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod install
```

### Report

* What did you do?

* What did you expect to happen?

* What happened instead?


### Stack

```
   CocoaPods : 1.15.2
        Ruby : ruby 3.3.5 (2024-09-03 revision ef084cc8f4) [x86_64-darwin24]
    RubyGems : 3.5.16
        Host : macOS 15.0.1 (24A348)
       Xcode : 16.0 (16A242d)
         Git : git version 2.39.5 (Apple Git-154)
Ruby lib dir : /Users/sedeam/.rbenv/versions/3.3.5/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'AppRKESP32' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for AppRKESP32
  pod 'FirebaseAnalytics'
  pod 'FirebaseAuth'
  pod 'FirebaseFirestore'
  pod 'GoogleSignIn'

end
```

### Error

```
ArgumentError - [Xcodeproj] Unable to find compatibility version string for object version `77`.
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/xcodeproj-1.25.1/lib/xcodeproj/project.rb:85:in `initialize'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/project.rb:48:in `initialize'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer/xcode/pods_project_generator/project_generator.rb:82:in `new'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer/xcode/pods_project_generator/project_generator.rb:82:in `create_project'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer/xcode/pods_project_generator/project_generator.rb:73:in `generate!'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer/xcode/single_pods_project_generator.rb:16:in `generate!'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer.rb:320:in `block in create_and_save_projects'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/user_interface.rb:64:in `section'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer.rb:315:in `create_and_save_projects'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer.rb:307:in `generate_pods_project'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer.rb:183:in `integrate'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/installer.rb:170:in `install!'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command/install.rb:52:in `run'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/lib/cocoapods/command.rb:52:in `run'
/Users/sedeam/.rbenv/versions/3.3.5/lib/ruby/gems/3.3.0/gems/cocoapods-1.15.2/bin/pod:55:in `<top (required)>'
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod:25:in `load'
/Users/sedeam/.rbenv/versions/3.3.5/bin/pod:25:in `<main>'

**Comment by eomonteiro** [comment]
try
platform :ios, '9.0' to platform :ios, '13.0' or '14.0'

**Comment by Marquis103** [comment]
Will there be an update to support `PBXFileSystemSynchronizedRootGroup`? Converting to a group is not actually solving the issue. It would be good to be able to support this type so that we can use this group in our app.

**Comment by LuisMoraEL** [comment]
No solution worked for me, I had to create a new project and use the XCode dependency manager, hopefully in the future they will fix it, but now I don't have a sure solution to share you :(

**Comment by khaled2024** [comment]
> Please follow the below steps to fix this issue
> 
> Step 1: Right click project Folder in Xcode and convert to group
> 
> Step 2: right click projectname.xcodeproj file and select Show Package Contents you can see the file project.pbxproj and open with texteditor
> 
> Remove the following lines in this file minimizedProjectReferenceProxies = 1; preferredProjectObjectVersion = 77;
> 
> Also update the below version objectVersion = 77; to objectVersion = 56;
> 
> Now you can pod init and install with your project

go here :-
https://stackoverflow.com/questions/76590131/error-while-build-ios-app-in-xcode-sandbox-rsync-samba-13105-deny1-file-w

**Comment by dppeak** [comment]
See my comment here.  https://github.com/CocoaPods/CocoaPods/issues/12618#issuecomment-2433155830

**Comment by p0w1nd** [comment]
Same here. Hope there's a fix soon. Thanks!

**Comment by amorde** [comment]
This is fixed in 1.16.0

**Comment by felipesantos17** [comment]
https://github.com/CocoaPods/Xcodeproj/issues/996
